specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/webapplicationfirewall.json

```diff
@@ -453,13 +453,28 @@
           "type": "boolean",
           "description": "Whether to allow WAF to check request Body."
         },
+        "reqBodyInspectLimitInKB": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 8,
```

Comment:
Why set the minimum to 8? Would it be valid to only inspect 1KB?

Reply:
@eschwabe We don't want to change any existing customer behavior at this point and so we want the keep the minimum limits the same as what they were in the previous configuration.